packages/generator/templates/app/.env.local

```diff
@@ -1,8 +1,9 @@
 # This env file should NOT be checked into source control
 # This is the place for values that changed for every developer
 
-# SQLite is ready to go out of the box, but you can switch to Postgres
-# by first changing the provider from "sqlite" to "postgres" in the Prisma
-# schema file and by second swapping the DATABASE_URL below.
-DATABASE_URL="file:./db.sqlite"
-# DATABASE_URL=postgresql://__username__@localhost:5432/__name__
+if (process.env.database === 'sqlite') {
+DATABASE_URL=__databaseUrl__
+# DATABASE_URL=postgresql://postgres@localhost:5432/__name__
+} else {
+DATABASE_URL=__databaseUrl__
+}
```

Comment:
@aem this if/else template syntax isn't working here. Do you know why?

Reply:
we can only use the if/else syntax in `.(t|j)sx?` files right now because it requires parsing the AST. I can try to rig something up with basic string matching for non-JS/TS files but it'll take a bit

Reply:
Could this be parsed as a shell script? 🤔